### ISSUE TITLE
Queue file handling

### ISSUE THREAD

**Issue by OricMuso** [Beep6581/RawTherapee#3653]
The example of how to enter code to get the output file doesn't work. Forward slash is used perhaps instead of comma ? The default which should put in folder called 'converted' doesn't go to the right address and put the pictures in the RawTherapee folder.
The %s1 to put a sequential number after the file name worked for the first file. I stopped the processing and then started it again and the sequential numbering didn't happen. 
 
Should I be able to add something to the filename? For instance I want my processed DSC_0101.raw to out put to DSC_0101rt.jpg   as I'm shooting both raw and jpeg and already have a file DSC_0101.jpg.

**Comment by TooWaBoo** [comment]
I've tried and both examples worked without any issue.
%p1/converted/%f%s1
%p1/converted/%frt

**Comment by OricMuso** [comment]
Well it doesn't for me. Win7 system if that makes a difference.

**Comment by TooWaBoo** [comment]
I'm on Win 10. Can you post exactly the whole string you are using, please?

**Comment by OricMuso** [comment]
I used the default and tried adding /%s1 or ,%s1. Tried %p4 instead of %p1 with no change.

**Comment by heckflosse** [comment]
@OricMuso I repeat TooWaBoo's request:

Can you post exactly the whole string you are using, please?

**Comment by TooWaBoo** [comment]
Don't use "/" or "," only add %s1 or rt

**Comment by OricMuso** [comment]
Think I've got it working now. The Use template had become unclicked for some reason so it was saving to folder (none).  Then it was naming the folder instead of giving the photo the filename.  It's working with rt suffix too I think I tried a space or +rt or something. 

So / is used for file structure and no comma (which was in the instructions. ) Thanks for help. 

(I use old computers that had DOS commands that used commas etc. Old habits !)
